src/deploy_inst.h

```diff
@@ -81,6 +82,28 @@ class DeployInst : public cyclus::Institution {
     "uilabel": "Lifetimes" \
   }
   std::vector<int> lifetimes;
+
+ private:
+#pragma cyclus var { \
+    "default": 0.0, \
+    "uilabel": "Geographical latitude in degrees as a double", \
+    "doc": "Latitude of the agent's geographical position. The value should " \
+           "be expressed in degrees as a double." \
+}
+double latitude;
+
+#pragma cyclus var { \
+    "default": 0.0, \
+    "uilabel": "Geographical longitude in degrees as a double", \
+    "doc": "Longitude of the agent's geographical position. The value should " \
+           "be expressed in degrees as a double." \
+}
+double longitude;
+
+cyclus::toolkit::Position coordinates;
+
+  /// Records an agent's latitude and longitude to the output db
+void RecordPosition();
```

Comment:
GCSG: this should be indented like other functions

Reply:
Thank you. I'll update the indentation on the file.